packages/relay/tests/lib/repositories/hbarLimiter/ethAddressHbarSpendingPlanRepository.spec.ts

```diff
@@ -62,7 +62,7 @@ describe('EthAddressHbarSpendingPlanRepository', function () {
       });
 
       this.afterAll(async () => {
-        await redisInMemoryServer.stop();
+        // await redisInMemoryServer.stop();
```

Comment:
Why is this commented out? If not needed we can remove it.

Reply:
It is needed, otherwise this server will keep running for all tests.

Please, revert this change.

Reply:
Sorry, I had failed to push the commit that fixes this